Jenkinsfile

```diff
@@ -234,8 +234,8 @@ node {
       'car tests': {
         pcStage("car tests") {
           sh label: "build", script: "selfdrive/manager/build.py"
-          sh label: "test_models.py", script: "INTERNAL_SEG_CNT=250 INTERNAL_SEG_LIST=selfdrive/car/tests/test_models_segs.txt FILEREADER_CACHE=1 \
-              pytest selfdrive/car/tests/test_models.py"
+          sh label: "test_models.py", script: "FILEREADER_CACHE=1 RANDOM_SEED=$RANDOM INTERNAL_SEG_CNT=250 \
```

Comment:
why is the seed generated outside the test?

Reply:
pytest x-dist re-runs the entire file to gather tests, so something like `random.randint` even globally will produce different values for each worker. `pytest-randomly` by default sets the seed to a fixed value for each worker, so I added that instead: https://github.com/pytest-dev/pytest-randomly#features